utils/conditions/conditions.go

```diff
@@ -23,32 +18,12 @@ import (
 )
 
 const (
-	// Verify Spec constants
-
 	// InvalidSpecReason indicates that the spec is invalid
 	InvalidSpecReason = "InvalidSpec"
 	// MissingFieldMessage the message to indicate rollout is missing a field
 	MissingFieldMessage = "Rollout has missing field '%s'"
 	// RolloutSelectAllMessage the message to indicate that the rollout has an empty selector
 	RolloutSelectAllMessage = "This rollout is selecting all pods. A non-empty selector is required."
```

Comment:
I'm wondering if we still need this consts since we move these checks to validation